README.md

```diff
@@ -1,410 +1,61 @@
-| `instance_hdd_size` |                                          | 10                                                                        | Sets the instance hard disk size in GB                                                                                                                                                                                                                                                           |
-| `instance_type`     | `m`, `l`, `xl`                           | `m`                                                                       | Sets the instance CPU size. You can also specify vendor specific machines in AWS i.e. `t2.micro`. [See equivalences](#Supported-vendors) table below.                                                                                                                                            |
-| `instance_gpu`      | ``, `tesla`, `k80`                       | ``                                                                        | Selects the desired GPU for supported `instance_types`.                                                                                                                                                                                                                                          |
-| `ssh_private`       |                                          |                                                                           | An SSH private key in PEM format. If not provided, one private and public key wll be automatically generated and returned in `terraform.tfstate`                                                                                                                                                 |
-
-### Machine
+- Rapidly move local machine learning experiments to a cloud infrustructure
+- Take advantage of training models on spot instances without losing any progress
+- Configure provisioning of compute resources from any of the supported vendors in a unified manner
+- Automatically unregister and remove cloud resources (never forget to turn your GPU off again)
 
-Setup instructions:
+## Prerequisites
 
-1. Setup your provider credentials as ENV variables
+To use Iterative Provider you will need to:
 
-<details>
-<summary>AWS</summary>
-<p>
+- [Install](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) Terraform 1.0+.
+- Make sure you have an account with a cloud vendor of choice, and have the respective authentication credentials saved into environment variables. Check out cloud-specific authentication method details in the [docs](<(https://registry.terraform.io/providers/iterative/iterative/latest)>)
 
-```sh
-export AWS_SECRET_ACCESS_KEY=YOUR_KEY
-export AWS_ACCESS_KEY_ID=YOUR_ID
-```
+Iterative Provider can provision resources with the following vendors and systems:
```

Comment:
```suggestion
Iterative Provider can provision resources with the following cloud providers and orchestrators:
```

“Cloud provider” is, [by far](https://books.google.com/ngrams/graph?content=cloud+provider%2Ccloud+vendor&year_start=2000&year_end=2019&corpus=26&smoothing=1&case_insensitive=true) more popular than “cloud vendor”, although the former might be a bit ambiguous in a context where we also talk about [Terraform] providers.

Reply:
Yeah, that was my reasoning. We have Iterative Provider and cloud providers... Even though these are not the same thing, it feels repetitive. But I'll commit your suggestion.

Reply:
Yes, it's an unlucky coincidence. In fact, a Terraform Provider is intended to be an abstraction layer for a single Cloud Provider; _id est:_ [`terraform-provider-aws`](https://github.com/hashicorp/terraform-provider-aws) abstracts all the AWS resources.

Reply:
If you believe that we should use “vendors” instead, we should probably update all the documentation. 😅